frontend/src/utils/decode-data.ts

```diff
@@ -1,3 +1,16 @@
+const double_backslash_escaped_regex = /(\\x)([\da-f]{2})|(\\u)([\da-f]{4})/gi
+const no_backslash_escaped_regex = /(?<!\\)(u)([\da-f]{4})/gi
```

Comment:
Do we not need to have a similar case for `\x` characters?

Reply:
No, that kind of encoding/decoding error never happened :)